concepticondata/conceptlists/Blust-1981-200.tsv

```diff
@@ -70,7 +70,7 @@ Blust-1981-200-68	68	needle	1382	NEEDLE (FOR SEWING)
 Blust-1981-200-69	69	to hunt (for game)	1435	HUNT
 Blust-1981-200-70	70	to shoot (an arrow)	1172	SHOOT
 Blust-1981-200-71	71	to stab, pierce	1434	STAB
-Blust-1981-200-72	72	to hit (with stick, club)	353	STRIKE
+Blust-1981-200-72	72	to hit (with stick, club)	1433	HIT
```

Comment:
Same here - you surely can unintentionally hit someone with a stick or club, but it's not what was likely elicited.